### PR TITLE
Create migration and seed

### PR DESCRIPTION
**Pull request by at-phuctv**
- create migration
- create model
- create seed table categories and roles

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,4 +6,5 @@
 Homestead.json
 Homestead.yaml
 .env
+composer.lock
```
Comment (baoviet) [inline .gitignore]: Don't add `composer.lock` to `.gitignore` file

Comment (ghost) [inline .gitignore]: Don't add `composer.lock` to `.gitignore` file. You can push this file to git repo.

File: app/Models/BookRoom.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    */
+    protected $table = 'book_rooms';
+
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+    'id', 'user_id', 'post_id',
```
Comment (ghost) [inline app/Models/BookRoom.php]: Please indent this line

File: app/Models/BookRoom.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        return $this->belongsTo('App\Models\Post');
+    }
+    /**
+     * [relation with user model]
+     *
+     * @return [type] [table user]
+     */
+    public function user()
+    {
+        return $this->belongsTo('App\User');
```
Comment (ghost) [inline app/Models/BookRoom.php]: Please add Model User to namespace `App\Models\`

File: app/Models/BookRoom.php [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+     *
+     * @var array
+     */
+    protected $fillable = [
+    'id', 'user_id', 'post_id',
+    ];
+    /**
+     * [relation with post model]
+     *
+     * @return [type] [table post]
```
Comment (ghost) [inline app/Models/BookRoom.php]: What is `[type] [table post]`
Please note correct data return. Same with another document comment

File: app/Models/Category.php [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'id', 'name', 'introduce', 'image',
+    ];
+    /**
+     * [relation with post model]
```
Comment (ghost) [inline app/Models/Category.php]: Don't add example description.

**Pull request merged**